test/fgpu/test_ddc.py

```diff
@@ -115,6 +115,7 @@ def test_bad_template_parameters(context: AbstractContext, taps: int, subsamplin
 
 
 def test_bad_tuning(context: AbstractContext) -> None:
+    """Test that :class:`DDCTemplate` raises ValueError when given bad tuning parameters."""
```

Comment:
Not necessarily a problem for you to solve right now, but I have to ask
* How are you/we ensuring this `:class:<...>` syntax is working when directly referencing the module?

That is, looking at files around it, there is a range of syntax used. Fair enough, it's consistent in the file.
* I checked on the Module Documentation and didn't see the tests at the tail end of the PDF

Anyway, perhaps a candidate for some technical debt.

![image](https://github.com/user-attachments/assets/55e291ad-60cc-44fd-8511-34d398069e26)